src/api/login.js

```diff
@@ -1,11 +1,25 @@
-import { BODY_OK, BODY_ERROR } from 'test/mocks/login';
+import { makeRequest, METHODS } from 'api/apiManager';
+import { BODY_OK } from 'test/mocks/login';
 
-const login = (username, password) => new Promise((resolve, reject) => {
-  if (username && password) {
-    resolve(BODY_OK);
-  } else {
-    reject(BODY_ERROR);
-  }
+export const getErrors = (username, password) => (
+  (username === 'admin' && password === 'adminadmin') ?
+    [] :
+    [{ code: 1, message: 'invalid credentials' }]
+);
+
+const login = (username, password) => makeRequest({
+  uri: '/entando-sample/OAuth2/access_token',
```

Comment:
Probably `/entando-sample/` should not be placed here. Should the right place be the `DOMAIN` variable in `.env` file?

Reply:
I'm torn on this one:
the attribute is named `DOMAIN` and that is not part of it.
On one hand the repetition would be endless tho.

I can make the change if you want to, though

Reply:
The problem is we'll have many server instances, and their URI will unlikely begin with `entando-sample`, so I think it's a good idea to have a neutral API path here, and set domain and path in the .env file.

You are right that's not part of the domain, so we could either rename DOMAIN in something like API_URI, or add another variable for the path. What do you think?